Refresh.GameServer/Endpoints/Game/CommentEndpoints.cs

```diff
@@ -82,6 +88,12 @@ public Response PostLevelComment(RequestContext context, GameDatabaseContext dat
         GameLevel? level = database.GetLevelByIdAndType(slotType, id);
         if (level == null) return NotFound;
 
+        
+        int count = database.GetNotificationCountByUser(level.Publisher);
+        if (!level.Publisher.Equals(user) && count < 10) {
```

Comment:
Also, we use [BSD braces](https://en.wikipedia.org/wiki/Indentation_style#Allman_style)